Prace_domowe/Praca_domowa3/Grupa1/Kacper_Grzymkowski/PD3.ipynb

```diff
@@ -0,0 +1,870 @@
+{
```

Comment:
<p>Doceniam, że próbuje Pan stosować modele, które nie były omawiane na laboratorium. Tego konkretnego modelu, nigdy nie używałam. Z tego co udało mi się wyczytać, to rzeczywiście w przypadku tego modelu często pojawia się problem z pamięcią. Być może istnieją inne implementacje, które są na to odporne. W przypadku ustawienia parametru copy na False teoretycznie problem z pamięcią, rzeczywiście powinien być mniejszy, ale tutaj to nie pomogło - zreprodukowałam ten problem. Wydaje się, że sklearn w tym przypadku tworzy macierz (NxN) gdzie N to liczba obserwacji aby wyznaczyć kowariancję.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/162/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>